hydra/app/views/catalog/show.html.erb

```diff
@@ -30,4 +30,16 @@
 
 <div id="sidebar" class="col-md-3 col-sm-4">
    <%= render_document_sidebar_partial %>
-</div>
\ No newline at end of file
+  <div id="csl-citation">
+      <h4>Chicago citation style</h4>
+    <span class="chicago-citation"><%= ChicagoCitationService.new.format(@document, request.original_url) %></span>
```

Comment:
Minor refinement; consider one of two options:

1. `ChicagoCitationService.format(document: @document, original_url: request.original_url)`
2. `ChicagoCitationService.new(document: @document, original_url: request.original_url).format`

I would expect to pass the "initial" values to either initialize or a single class method.  This narrows the interface of the class; and in option 1 could move to a module which has nominal memory improvements.  (Instantiating a class comes with a minor memory hit and garbage collection hit).

Otherwise, this is a fantastic feature!